src/Psalm/Internal/Analyzer/MethodComparator.php

```diff
@@ -113,6 +115,29 @@ public static function compare(
             );
         }
 
+        if (!$guide_classlike_storage->user_defined
+            && $implementer_classlike_storage->user_defined
+            && $codebase->analysis_php_version_id >= 80100
```

Comment:
Using 8_01_00 didn't work?

Reply:
Turns out it's invalid syntax on PHP 7.1

Reply:
right, forgot this was a thing and we only did the change on Psalm 5